File: Programmers/77884.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

bool isEven(int number) {
    int cnt = 0;

    for(int i = number; i >= 1; i--) {
        if(number%i == 0)
            cnt++;
    }

    if (cnt%2 == 0)
        return true;
    else
        return false;
}

int solution(int left, int right) {
    int answer = 0;

    for(int i = left; i <= right; i++) {
        if(isEven(i)) {
            answer += i;
        } else {
            answer -= i;
        }
    }

    return answer;
}

int main() {
    int left = 24;
    int right = 27;
    cout << solution(left, right) << endl;
}

File: Python/77884.py
def countY(num):
    count = 0
    for i in range(1, num + 1):
        if num%i == 0:
            count += 1
    return count

def solution(left, right):
    answer = 0
    for i in range(left, right + 1):
        if countY(i)%2 == 0:
            answer += i
        else:
            answer -= i
    return answer